src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-size: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.logo__image {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
This styles wasn't applied, you have a mistake in class name
Also they are unnecessary and can be removed